cicd/git-version.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+cd $(dirname $0)/../src/environments
+
+export GIT_TAG=$(git tag -l --points-at HEAD)
+export GIT_REVISION=$(git rev-parse --short HEAD)
+
+envsubst '${GIT_TAG} ${GIT_REVISION}' \
```

Comment:
@hall I saw `git.version.ts` that landed in `master` - thing is these Docker images don't have `git`, and typically `.git` is a in `.dockerignore` anyways.  So I moved it to a shell script and CI runs the shell script now.  This approach work?